Formula/y/yass-cli.rb

```diff
@@ -0,0 +1,84 @@
+class YassCli < Formula
+  desc "Lightweight and efficient, socks/http forward proxy"
+  homepage "https://letshack.info"
+  url "https://github.com/Chilledheart/yass/releases/download/1.13.2/yass-1.13.2.tar.bz2"
+  sha256 "1b661be8901c0c8c8bf292b02563ca8abdba503a9f1f7de8ca5f808aff63e8a4"
+  license "GPL-2.0-only"
+  head "https://github.com/Chilledheart/yass.git", branch: "develop"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```
If the problem is picking up weird versions, then we should use a `regex` instead of `:github_latest`.

Reply:
I don't see any problem we should need to resolve here.

Reply:
That isn't the suggestion. We don't want to use the `github_latest` strategy because it uses the API and we prefer not to do that.
If you use the `regex` provided in the suggestion above it will only match versions that use your current version scheme.

Reply:
It is the different from git's tags and GitHub releases where GitHub releases depend on git tags but not the vice versa.

Yass currently uses GitHub as the main git host site. It makes sense if yass releases other releases without GitHub release.

However, it is possible that some tags are pushed but releases are not ready yet. It would be convenient if fixes are push before releases are finally out.

And there are two release lines which are stable release and legacy release. Both of them use the same point release style such as x.y.z. It would be inconvenient if regex cannot tell them from git tag name. Both of these new releases are pushed regularly in the life of time. However newest GitHub release will always be stable releases.

Reply:
Thank you for the explanation, using `github_latest` makes sense in this instance.